src/Contact/View/page.twig

```diff
@@ -1,5 +1,23 @@
 {% include 'Common/View/header.twig' %}
 <section class="container">
-        <h1>Kontakt</h1>
+        <form method ="post" action="">   
```

Comment:
Przed i po = warto unikać spacji ;)
`method="post"`

Plusik za "post" - przy formularzach tego typu jest to bardzo dobry typ przesyłu danych ;) 
Innym możliwym jest get